install.sh

```diff
@@ -16,8 +16,8 @@ function display_help()
     echo "       --fast                  Quick-build RCCL (local gpu arch only, no backtrace support)"
```

Comment:
This line should be updated to reflect the options that occur when fast is picked

Reply:
This hasn't been fixed still